jaq-interpret/src/val.rs

```diff
@@ -109,6 +109,15 @@ impl Val {
         }
     }
 
+    /// If the value is a Str, extract the inner string, else convert
+    /// it to string.
+    pub fn to_string_or_clone(self) -> String {
+        match self {
+            Self::Str(s) => s.to_string(),
```

Comment:
Here I would use to_owned().
Otherwise, this will make people (like me ^^) reading this wonder whether this will escape the string.

Reply:
I'll have to go with `(*s).clone()` since `to_owned()` over an `Rc` yields another `Rc`. I think.

Reply:
That sounds good to me!